foreign/python/Cargo.toml

```diff
@@ -32,9 +32,12 @@ pyo3-async-runtimes = { version = "0.25.0", features = [
     "attributes",
     "tokio-runtime",
 ] }
-pyo3-stub-gen = "0.11.1"
+pyo3-stub-gen = "0.12.0"
 tokio = "1.40.0"
 
+[patch.crates-io]
+pyo3-stub-gen = { git = "https://github.com/Jij-Inc/pyo3-stub-gen.git" }
```

Comment:
Hm, I should lock down the git ref to a specific commit, reproducible builds and all

Reply:
Yes, it should be a reproducible build.